docs/book/deployment.md

```diff
@@ -168,11 +170,101 @@ files will be marked public on upload.
 See: [Cross-Origin Resource Sharing (CORS)](
   https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html)
 
+##### Upload Parameters
+
+You can specify S3 upload parameters to groups of your assets by utilizing the
+optional `s3Configuration.uploadParams` object list.
+
+A Litexa config that utilizes `s3Configuration.uploadParams` might look like:
+
+```javascript
+const deploymentConfiguration = {
+  name: 'my-skill',
+  deployments: {
+    production: {
+      module: '@litexa/deploy-aws',
+      s3Configuration: {
+        bucketName: 'my-skill-bucket',
+        uploadParams: [
+          {
+            filter: ['*.mp3'],
+            params: {
+              // check for file change every 10 minutes
+              // (useful for content files that are regularly updated)
+              CacheControl: 'max-age=600'
+            }
+          },
+          {
+            filter: ['*.jpg', '*.png'],
+            params: {
+              // always check for file change
+              // (useful during development)
+              CacheControl: 'no-cache'
+            }
+          },
+          {
+            params: {
+              // no file filter -> applies 1 hour age
+              // to all files that aren't caught by the
+              // above 2 filters
+              CacheControl: 'max-age=3600'
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+##### S3 Configuration Schema
+
+The schema for the `s3Configuration` object is as follows:
+
+* `bucketName` - (String)
+  * The name of the bucket that your assets are deployed to.
+* `uploadParams` - (Array\<Object\>) - Optional
+  * `filter` - (Array\<String\>) - Optional
+    * A list of glob patterns that, when matched to your assets,
+      the upload params are applied to.
+      * Litexa uses the
+      [minimatch NPM package](https://www.npmjs.com/package/minimatch)
+      to implement the file pattern matching.
+  * `params` - (Object)
+    * An object that is passed into Litexa's call on
+      the AWS SDK S3 Client's `#upload()` function.
+      * To understand what keys are acceptable in the
+      `params` object, read more about the AWS SDK S3 Client's
+      `#upload()` function in the
+      [AWS SDK docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property).
+
+:::tip You can define default upload params
+An `uploadParams` object that either has no `filter` specified, or includes a
+`'*'` filter, will be treated as a default set of upload parameters. All
+assets that do not match any other filters will use these `params`.
+:::
+
+:::tip The order of upload params matters
+Any `uploadParams` objects that are not default upload parameters (see above)
+are applied in order. This means that individual assets will be set to use the first
```

Comment:
may want to point out that it uses *only* the first matching filter's params, not that they are applied for every filter they pass (settings are not cumulative)

Make sure that comments here and in the deploy module's readme are applied to both locations.